src/components/learning-path/LearningPathHeaderSelect.tsx

```diff
@@ -0,0 +1,129 @@
+import { useEffect, useRef, useState } from 'react'
+import { FaIcon } from '../ui/FaIcon'
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons'
+import clsx from 'clsx'
+
+export default function LearningPathHeaderSelect({
+  options,
+  onSelect,
+}: {
```

Comment:
I'm mostly using a separate interface like `LearningPathHeaderSelectProps` in this situation